tha2/app/ifacialmocap_puppeteer.py

```diff
@@ -421,9 +435,14 @@ def load_image(self, event: wx.Event):
             image_file_name = os.path.join(file_dialog.GetDirectory(), file_dialog.GetFilename())
             try:
                 pil_image = resize_PIL_image(extract_PIL_image_from_filelike(image_file_name))
-                w, h = pil_image.size
-                self.wx_source_image = wx.Bitmap.FromBufferRGBA(w, h, pil_image.convert("RGBA").tobytes())
-                self.torch_source_image = extract_pytorch_image_from_PIL_image(pil_image).to(self.device)
+                w, h, c = pil_image.size

```

Comment:
Change to w,h = pil_image.size. For a PIL image, size is a 2-tuple, not a 3-tuple. (See https://pillow.readthedocs.io/en/4.1.x/reference/Image.html#PIL.Image.size)